packages/es-components/src/components/controls/checkbox/Checkbox.js

```diff
@@ -117,12 +142,15 @@ function Checkbox({
       <CheckboxText className="es-checkbox__text" aria-hidden={!!ariaLabel}>
```

Comment:
While I was here, I figured I would ask, do we really want to use aria-hidden if we don't receive an ariaLabel prop?

Reply:
I guess the intent was if the aria-label is supplied the screen reader would read that instead of the checkbox text. I don't know if this is a sensible pattern or not. Seeing as `labelText` is required, there should always be text associated with the checkbox for a screenreader to read anyway. I'd vote remove it personally.